diff --git a/README.md b/README.md
index 316d837..b6f9bc2 100644
--- a/README.md
+++ b/README.md
@@ -39,6 +39,8 @@ cd isaaclab/IsaacLab
 # UR5e Scale of 0.1
 source isaaclab/bin/activate
 cd /home/jofa/Downloads/Isaac_Lab_UR5e_Reach
-wandb sweep --project rel_ik_sb3_ppo_ur5e_reach_0_1 config_0_1.yaml
+wandb sweep --project rel_ik_sb3_ppo_ur5e_reach_0_1_corrected config_0_1.yaml
 wandb agent jofan23-university-of-southern-denmark/rel_ik_sb3_ppo_ur5e_reach_0_1/za203z2j
-wandb agent jofan23-university-of-southern-denmark/rel_ik_sb3_ppo_ur5e_reach_0_1_v2/cs6czrhy
\ No newline at end of file
+wandb agent jofan23-university-of-southern-denmark/rel_ik_sb3_ppo_ur5e_reach_0_1_v2/cs6czrhy
+
+wandb agent jofan23-university-of-southern-denmark/rel_ik_sb3_ppo_ur5e_reach_0_1_corrected/s5h00sc3
\ No newline at end of file
diff --git a/config_0_1.yaml b/config_0_1.yaml
index cbda6f0..9f04543 100644
--- a/config_0_1.yaml
+++ b/config_0_1.yaml
@@ -1,6 +1,6 @@
 program: train_sb3_wandb.py
 method: grid
-name: rel_ik_sb3_ppo_ur5e_reach_0_1_v2
+name: rel_ik_sb3_ppo_ur5e_reach_0_1_corrected
 metric:
   goal: minimize
   name: train/loss
diff --git a/gym_env/env/reach_env_cfg.py b/gym_env/env/reach_env_cfg.py
index 4cfa3e3..dba0d6a 100644
--- a/gym_env/env/reach_env_cfg.py
+++ b/gym_env/env/reach_env_cfg.py
@@ -215,8 +215,8 @@ class ObservationsCfg:
         # joint_vel = ObsTerm(func=mdp.joint_vel_rel, noise=Unoise(n_min=-0.01, n_max=0.01))
 
         # gripper_joint_pos = ObsTerm(func=mdp.joint_pos, params={"asset_cfg": SceneEntityCfg("robot", joint_names=["joint_left", "joint_right"]),},)
-        tcp_pose = ObsTerm(func=mdp.get_current_tcp_pose)
-        pose_command = ObsTerm(func=mdp.generated_commands, params={"command_name": "ee_pose"})
+        tcp_pose = ObsTerm(func=mdp.get_current_tcp_pose) # TCP pose in base frame
+        pose_command = ObsTerm(func=mdp.generated_commands, params={"command_name": "ee_pose"}) # Desired ee (or tcp) pose in base frame
         actions = ObsTerm(func=mdp.last_action)
 
         def __post_init__(self):
diff --git a/train_sb3_wandb.py b/train_sb3_wandb.py
index 771a3d3..b7be766 100644
--- a/train_sb3_wandb.py
+++ b/train_sb3_wandb.py
@@ -39,7 +39,7 @@ def main():
         config = yaml.load(file, Loader=yaml.FullLoader)
 
     run = wandb.init(
-        project="rel_ik_sb3_ppo_ur5e_reach_0_1",
+        project="rel_ik_sb3_ppo_ur5e_reach_0_1_corrected",
         config=config,
         sync_tensorboard=True,  # auto-upload sb3's tensorboard metrics
         monitor_gym=False,  # auto-upload the videos of agents playing the game
